config/config/wc-config.yaml

```diff
@@ -382,17 +382,13 @@ issuers:
   ## Deploy let's encrypt ACME issuers
   ## "letsencrypt-prod" and "letsencrypt-staging".
   letsencrypt:
-    enabled: true
-
-    ## Namespaces in which to install the letsencrypt issuers in.
-    namespaces: []
-
-    prod:
-      ## Mail through which letsencrypt can contact you.
-      email: "set-me"
-    staging:
-      ## Mail through which letsencrypt can contact you.
-      email: "set-me"
+    enabled: false
+    # prod:
+    #   ## Mail through which letsencrypt can contact you.
+    #   email: null
```

Comment:
Just tested and this did not work as expected. Using `""` instead gives a warning.

Reply:
With 
```ShellSession
issuers:
  letsencrypt:
    prod:
      email: ""
    staging:
      email: null
```

yq returns nothing/null in bash with `email: ""`, whereas it return the string `null` when `email: null` :facepalm:
Leaving the value empty for a field also works `email:` as yq will return empty string.

Reply:
Oh good to know. I'll change the comment to have `""` instead of `null` then.

Reply:
This bothers me as all of these should be equivalent as according to the [YAML specification](https://yaml.org/spec/1.2/spec.html#id2805071)
```ShellSession
Regular expression         Resolved to tag
null | Null | NULL | ~     tag:yaml.org,2002:null
/* Empty */                tag:yaml.org,2002:null
```
```
parent:
  key1:               # empty so "null", # is a comment!
  key2: ~             # also "null"
  key3: null          # "null" explicitly ))
  key4: !!null "null" # for the funs of "secondary tag handle: !!"
  key5: "null"        # sorry, it is a string or !!str if you like ((
```